pom.xml

```diff
@@ -2471,6 +2471,7 @@
         <module>cdap-test</module>
         <module>cdap-integration-test</module>
         <module>cdap-ui</module>
+        <module>cdap-kms</module>
```

Comment:
move this up. It has to be before so that it gets compiled before cdap-master (since there is no dependencies) so that in the packaging plugin in cdap-master, we can copy the jar.